utils/http.go

```diff
@@ -0,0 +1,64 @@
+package utils
+
+import (
+  "strings"
+  "errors"
+  "net/http"
+  "io"
+  "os"
+)
+
+
+func GetRawURL(url string) string {
+	if strings.Contains(url, "github.com") {
+		raw:=strings.Replace(url, "github.com", "raw.githubusercontent.com", 1)
+		return strings.Replace(raw, "/blob", "", 1)
+	}
+	return url
+}
+
+func IsGitHubURL(url string) bool {
+	return strings.Contains(url, "github")
+
+}
+
+func GetFileName(url string) string {
```

Comment:
This should probably use https://pkg.go.dev/path#Base.

Reply:
actually this is also available under utils/file.go -> `FilenameForURL`
https://github.com/srl-labs/containerlab/blob/main/utils/file.go#L259

Reply:
updated to use FilenameForURL.